skrub/tests/test_multi_agg_joiner.py

```diff
@@ -0,0 +1,436 @@
+import pandas as pd
+import pytest
+from pandas.testing import assert_frame_equal
+
+from skrub._dataframe._polars import POLARS_SETUP
+from skrub._multi_agg_joiner import MultiAggJoiner
+
+
+@pytest.fixture
+def main():
```

Comment:
So for this file, I will have the exact same comment than for the `test_agg_joiner`.